--- a/tools/CMakeLists.txt	2026-09-29 20:39:38.147600772 +0200
+++ b/tools/CMakeLists.txt	2026-09-29 20:39:38.151600653 +0200
@@ -36,12 +36,11 @@
     if(QT_FEATURE_qml_preview)
         add_subdirectory(qmlpreview)
     endif()
-    if(QT_BUILD_SHARED_LIBS AND TARGET Qt::Quick)
+    if(TARGET Qt::Quick)
         add_subdirectory(qmlscene)
         add_subdirectory(qmltime)
     endif()
-    if(QT_BUILD_SHARED_LIBS
-        AND QT_FEATURE_process
+    if(QT_FEATURE_process
         AND QT_FEATURE_regularexpression
         AND TARGET Qt::Quick)
         add_subdirectory(qmlplugindump)
