--- a/configure	2023-09-22 23:10:41.000000000 +0200
+++ b/configure	2024-10-01 23:04:48.084869070 +0200
@@ -10967,16 +10967,11 @@
 
 if test "x$UUCPLOCK" != x
 then
-	if test -d $UUCPLOCK
-	then
 		{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $UUCPLOCK" >&5
 printf "%s\n" "$UUCPLOCK" >&6; }
 
 printf "%s\n" "#define UUCPLOCK \"$UUCPLOCK\"" >>confdefs.h
 
-	else
-		UUCPLOCK=""
-	fi
 fi
 
 if test "x$UUCPLOCK" = x
